CHANGELOG.rst

```diff
@@ -5,8 +5,7 @@ CHANGELOG
 1.16.3
 ======
 
-* bug-fix: Local Mode: Allow support for SSH in local mode 
```

Comment:
Can't really remove it - it's already released. Please add a new record for the revert change in the next version.